src/set.rs

```diff
@@ -104,6 +104,12 @@ impl SysnoSet {
         (self.data[idx] & (1 << (bit % Self::WORD_WIDTH))) != 0
     }
 
+    /// Returns true if the set is empty. This is an O(n) operation as
+    /// it must iterate over the entire bitset.
```

Comment:
Technically, this is still `O(1)` because the number of set bits in the set doesn't affect how long this function takes. (The total number of syscalls is always constant.)

I still expect this to be a very fast operation due to cache locality and the relatively low number of 64-bit words that get iterated over.